osu.Game/Rulesets/UI/Playfield.cs

```diff
@@ -443,6 +452,25 @@ public DrawableSamplePool(ISampleInfo sampleInfo, int initialSize, int? maximumS
 
         #endregion
 
+        private void onNewResult(DrawableHitObject drawable, JudgementResult result)
+        {
+            // Not using result.TimeAbsolute because that might change and also there is a potential precision issue.
+            judgementResults.Push(new JudgementResultEntry(Time.Current, drawable.Entry.AsNonNull(), result));
```

Comment:
Do you mind elaborating on this? What might change? What precision issue?

Sounds to me like you may be referring to what I already found as https://github.com/ppy/osu/issues/17482#issuecomment-1079986449? If so, then using `Time.Current` as a local workaround and leaving `TimeAbsolute` as a footgun for the next person to blow their foot off with seems pretty wrong to me.

Reply:
Change of `HitObject.StartTime`.
Precision issue is the catastrophic cancellation in `TimeAbsolute = StartTime + (hit time - StartTime)` so it might be slightly different from the original `hit time`.
On a second thought it might be better to fix `TimeAbsolute` issue by storing the original hit time in it rather than duplicating here. Though that is a potential behavior change so I'll try to see if it breaks something.